iree/compiler/Dialect/Flow/Transforms/PassDetail.h

```diff
@@ -0,0 +1,33 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef IREE_COMPILER_DIALECT_FLOW_TRANSFORMS_PASS_DETAIL_H_
+#define IREE_COMPILER_DIALECT_FLOW_TRANSFORMS_PASS_DETAIL_H_
+
+#include "mlir/Pass/Pass.h"
+
+namespace mlir {
+namespace iree_compiler {
+namespace IREE {
+namespace Flow {
+
+#define GEN_PASS_CLASSES
+#include "iree/compiler/Dialect/Flow/Transforms/Passes.h.inc"
```

Comment:
So I know that MLIR never does this, but shouldn't we be undefing this macro after we're done with it?

```suggestion
#define GEN_PASS_CLASSES
#include "iree/compiler/Dialect/Flow/Transforms/Passes.h.inc"
#undef GEN_PASS_CLASSES
```

(same throughout)

Reply:
Actually, the undef happens consistently in the generated code, so not going to commit this one.

Reply:
Ah! That makes sense